library/src/main/java/com/bumptech/glide/request/RequestOptions.java

```diff
@@ -186,8 +189,8 @@ public static RequestOptions signatureOf(@NonNull Key signature) {
   public static RequestOptions fitCenterTransform() {
     if (fitCenterOptions == null) {
       fitCenterOptions = new RequestOptions()
-          .fitCenter()
-          .autoClone();
+              .fitCenter()
+              .autoClone();
```

Comment:
Looks like you may have a different formatter that the project. Glide uses 2-space indents and continuation is usually double indent, so 2*2=4, 8 is weird in this code base.